.travis.yml

```diff
@@ -10,6 +10,7 @@ services:
   - docker
 
 script:
+  - go test -v ./...
```

Comment:
This will most likely fail as the txtdirect plugin isn't yet linked. So it would need the linking as preparation step.

Reply:
I don't think the linking to caddy is actually needed for our tests, they run fine on their own

Reply:
At least they do on my machine